helper/helper.ts

```diff
@@ -1,15 +1,23 @@
+import { getEnv } from '@helper/environment';
 import { IconikService } from '@workflowwin/iconik-api';
+import { IconikContext } from '../authorizers/iconik/context';
 
-export function createIconikService(options?, request?) {
-  const iconikOptions = {
-    appId: process.env.ICONIK_APP_ID as string,
-    authToken: request ? request.headers['Auth-Token'] || request.headers['auth-token'] : process.env.ICONIK_TOKEN,
-    iconikUrl: process.env.ICONIK_URL as string,
-    systemDomainId: process.env.ICONIK_SYSTEM_DOMAIN_ID,
-    ...options,
-  };
+export function createIconikClient(authContext?: IconikContext): IconikService {
```

Comment:
I suggest to move out this to `helper/iconik.ts`. So, that module includes only iconik related helpers and that`s clear.

Reply:
Sounds good